### ISSUE TITLE
property has no __doc__

### ISSUE THREAD

**Issue by twoertwein** [microsoft/pyright#3227]
**Describe the bug**
`property` is most likely a sub-class of `object` and should therefore have `__doc__` (at least accessing `__doc__` at runtime works).

**To Reproduce**
```py
class Test:
    @property
    def test(self) -> int:
        """
        Return something
        """
        return 1

Test.test.__doc__
```

> error: Cannot access member "__doc__" for type "property"
> error: Type of "__doc__" is unknown (reportUnknownMemberType)
> 

**Expected behavior**
No error.

**Comment by erictraut** [comment]
This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.232, which I just published. It will also be included in the next release of pylance.
